metpy/plots/tests/test_skewt.py

```diff
@@ -67,6 +67,20 @@ def test_skewt_with_grid_enabled():
         SkewT()
 
 
+@pytest.mark.mpl_image_compare(tolerance=0., remove_text=True)
+def test_skewt_arbitrary_rect():
+    """Test placing the SkewT in an arbitrary rectangle."""
+    fig = plt.figure(figsize=(9, 9))
+    SkewT(fig, rect=(0.15, 0.35, 0.8, 0.3))
+    return fig
+
+
+@pytest.mark.xfail(strict=True)
+def test_skewt_subplot_rect_conflict():
+    """Test the subplot/rect conflict failure."""
+    SkewT(rect=(0.15, 0.35, 0.8, 0.3), subplot=(1, 1, 1))
```

Comment:
Close, but replace the `@pytest.mark.fail(strict=True)` with this inside the function:
```python
with pytest.raises(ValueError):
   SkewT(rect=(0.15, 0.35, 0.8, 0.3), subplot=(1, 1, 1))
```

Reply:
Ah, yeah. that looks like the correct structure. Not as familiar with pytest as I probably should be.